modules/govcms_ckan_media/plugins/visualisation/table.inc

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * @file
+ * Basic table visualisation (default).
+ */
+
+$plugin = array(
+  'title' => t('Table visualisation'),
+);
+
+/**
+ * Returns a renderable array that represents the block content.
+ */
+function govcms_ckan_media_table_view($file, $display, $config) {
+  $element = array();
+  $chart_class = 'ckan-table';
+  $parser = govcms_ckan_dataset_parser();
+  $response = govcms_ckan_client_request_records($file->resource_id);
+
+  // If failure, provide error message.
+  if ($response->valid !== FALSE) {
```

Comment:
Not sure if this is relevant, but http://docs.ckan.org/en/latest/api/#making-an-api-request talks about valid responses against ckan. The parser currently checks the http status code, but the docs seem to indicate it is better to check against the success key in the response. Doesn't form part of this PR anyway, but just noting.